using-timescaledb/update-db.md

```diff
@@ -68,14 +66,14 @@ After executing the command, the psql `\dx` command should show the latest versi
 
     Name     | Version |   Schema   |                             Description
 -------------+---------+------------+---------------------------------------------------------------------
- timescaledb | 0.5.0   | public     | Enables scalable inserts and complex queries for time-series data
+ timescaledb | 1.0.0   | public     | Enables scalable inserts and complex queries for time-series data
```

Comment:
Should this be `x.y.z` so that it always shows the latest version? @RobAtticus ?

Reply:
Wasn't sure if that worked on this page, but yes, updated.